components/x-podcast-launchers/src/PodcastLaunchers.jsx

```diff
@@ -0,0 +1,49 @@
+import { h, Component } from '@financial-times/x-engine';
+import generateAppLinks from './generate-app-links';
+import generateRSSUrl from './generate-rss-url';
+
+function mapConceptToAcastSeries (/** concept */) {
```

Comment:
I think we'd need to derive the acast series name from the podcast series concept

Reply:
and something something https://github.com/Financial-Times/n-concept-ids/blob/master/src/index.js

Reply:
you are much safer relying on the concept ID, that is less likely to change. we could maintain a mapping of concept IDs to list of acast urls within this component